shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -1,18 +1,16 @@
 package mlscript
 
-import mlscript.utils._, shorthands._
+import mlscript.codegen.Helpers._
+import mlscript.codegen._
+import mlscript.utils._
```

Comment:
The `import mlscript.utils._, shorthands._` pattern is everywhere in the code base. No reason to make it different here. It sort if implicitly emphasizes that the two imports are related.

Reply:
Fixed in eb9dd5f.